lib/rubyblok/helpers/storyblok_helper.rb

```diff
@@ -10,7 +10,8 @@ def rubyblok_content_tag(content)
   end
 
   def rubyblok_story_tag(slug)
-    content = get_story_via_api(slug)["content"].to_dot
+    content = get_story_via_cache(slug)["content"].to_dot
```

Comment:
It should be `get_story_via_cache` or `get_story_via_api` depending on Rubyblok.configuration.cached config option